metrics-datadog/src/main/java/org/coursera/metrics/datadog/DynamicTagsCallBack.java

```diff
@@ -0,0 +1,7 @@
+package org.coursera.metrics.datadog;
+
+import java.util.List;
+
+public interface DynamicTagsCallback {
```

Comment:
Please add a javadoc for this public interface. Mention that implementations must execute very quickly, as they block the reporter.